pycti/api/opencti_api_connector.py

```diff
@@ -10,6 +10,43 @@ class OpenCTIApiConnector:
     def __init__(self, api):
         self.api = api
 
+    def read(self, connector_id: str) -> Dict:
+        """Reading the connector and its details
+
+        :return: return all the connector details
+        :rtype: dict
+        """
+        self.api.app_logger.info("[INFO] Getting connector details ...")
+        query = """
+            query GetConnectors($id: String!) {
```

Comment:
minor comment: since we get only one connector, I suggest to rename the query name to `GetConnector` without the `s`. Or just `Connector`